__tests__/Filters.test.jsx

```diff
@@ -1,9 +1,15 @@
 // should be able to sort by most recent
 // should be able to sort by number of stars
-import App from '../client/src/components/App';
+import Filters from '../client/src/components/Filters';
 
-describe('App', () => {
+const wrapper = shallow(<Filters />);
+
+describe('<Filters /> rendering', () => {
```

Comment:
This test appears that it is checking that an 'h3' tag is being rendered in the Filter component vs the Filter component actually being rendered inside of App.  Might want to update test description.

Reply:
yes, these tests aren't implemented yet. This is just a skeleton for further development.

Reply:
awesome, thanks for the clarification.  Everything looks good =)